Datos

Clarissa0512 opened the pull request:
### Issues

- La sección datos para ver la cantidad de becados y algunas otras cosas mas. 

#53 

### Todos

_¿Qué queda por hacer?_

- Revisar

### Áreas impactadas

_¿Qué páginas o componentes están afectados?_

- datos.html y datos.css

### Pasos para reproducir

_¿Qué necesitamos hacer para probar?_

Clarissa0512 committed:
Agregando macro a teaser...

Aquí agrego macro a la pagina teaser.

Clarissa0512 committed:
Merge branch 'master' of https://github.com/cosola-macile/macile.org

Clarissa0512 committed:
Añadiendo sección datos

Aquí añado los archivos de html y css de datos.

Clarissa0512 committed:
Agregando los el archivo de datos a site.css

Aqui añado el archivo de css de datos a site.css

Clarissa0512 committed:
Merge branch 'master' of https://github.com/cosola-macile/macile.org

Clarissa0512 committed:
Merge branch 'master' of https://github.com/cosola-macile/macile.org

Clarissa0512 committed:
Añadiendo archivos de las sección fact

Aquí añado los archivos de la sección fact.

misscs commented on app/assets/styles/site.css:
```diff
@@ -19,3 +19,5 @@
 @import 'modules/team-block/team-block.css';
 @import 'modules/scholarship/scholarship-item.css';
+@import 'modules/datos/datos.css';
+@import 'views/scholarships/overview.css';
```
Vamos a organizar este archivo un poquito :) Como `scholarships overview` es parte de una página debemos empezar una nueva sección. L13 nos indica que este sección de importaciones es para `/* Módulos Locales  */`. 

Puedes añadir una nueva sección?  

```css
/* 
 * Páginas
 * Estilos para todas secciones de macile.org  
 **/

/* Scholarships/Becas */
---
```

misscs commented on app/modules/datos/datos.css:
```diff
@@ -0,0 +1,10 @@
+.fact-content {
+  text-align: center;
+  width: 20px;
+  height: 90px;
```
No podemos usar altura fija porque no es muy flexible

Clarissa0512 replied:
👍

misscs commented on app/modules/datos/datos.css:
```diff
@@ -0,0 +1,10 @@
+  margin-right: 70px;
+  margin-left: 65px;
+  margin-bottom: 105px;
+}
```
Recuerda que preferimos usar shorthand `margin: x x x x;` Si estás declarando más de dos debemos usar la manera corta

Clarissa0512 replied:
😅 si es verdad.

misscs commented on app/modules/datos/datos.html:
```diff
@@ -0,0 +1,8 @@
+{% macro datos(data) %}
+<section class="fact-content">
+
+  <h1 class="fact-title">{{ data.result }}</h1>
```
Debes seguir el patrón para nombrar los archivos de macros

Clarissa0512 replied:
👍

misscs commented on app/modules/teaser_news/teaser.macro.html:
```diff
@@ -0,0 +1,15 @@
+    </div>
+</article>
+{% endmacro %}
+
```
Me parece muy raro que este archivo esta en el PR.

Clarissa0512 replied:
Eso ví. Sucede cuando creo un nuevo PR sin siquiera hacer commit aparecen archivos que no me pertenecen. 😑

misscs commented on app/views/scholarships/index.html:
```diff
@@ -1,9 +1,25 @@
   <h1>{{ scholarships.intro }}</h1>
 
+  <header class="fact-header">
+    <p>En esta sección se enterarán de diversas cosas como por ejemplo la cantidad de becas que MACILE la ha otorgado a los estudiantes.</p>
```
El contenido no debe ser estático. Creo que tenemos contenido para esto en `yaml` no?

misscs commented on app/views/scholarships/index.html:
```diff
@@ -1,9 +1,25 @@
+
   <h1>{{ scholarships.intro }}</h1>
 
+  <header class="fact-header">
```
El `header` es parte de la section `overview-header`, no?

Clarissa0512 replied:
Perdón, pero no veo la sección overview-header 😕.

misscs commented on app/modules/datos/datos.html:
```diff
@@ -0,0 +1,8 @@
+  <p class="fact-text">{{ data.description }}</p>
+
+</section>
+{% endmacro %}
```
No estoy muy de acuerdo con el html aquí. Sería más fácil explicar sobre una llamada. Lunes después de standup, ok?

Clarissa0512 replied:
👍

Clarissa0512 replied:
Cambiar el nombre del macro a fact para que sea en ingles, por lo tanto cambiar las clases tambien.

misscs commented on app/views/scholarships/index.html:
```diff
@@ -1,9 +1,25 @@
+
   <h1>{{ scholarships.intro }}</h1>
 
+  <header class="fact-header">
```
Puedes? centralizar el texto en `fact-header`? Gracias!

Clarissa0512 replied:
👍

misscs commented on app/views/scholarships/overview.css:
```diff
@@ -0,0 +1,12 @@
+
+.fact-header {
+  border-bottom: 1px solid rgb(100,100,90);
+  border-top: 1px solid rgb(100,100,90);
```
Y remover este border. 🙏

misscs reviewed (changes requested):
Gran trabajo completando esto! Algunas de mis sugerencias son cosas que me gustaría verte comenzando a encontrar. Tómese el tiempo necesario para revisar su trabajo :)

misscs commented on app/modules/datos/datos.css:
```diff
@@ -0,0 +1,10 @@
+.fact-content {
+  text-align: center;
+  width: 20px;
```
Tampoco no queremos utilizar width con flexbox items.

misscs commented on app/views/scholarships/overview.css:
```diff
@@ -0,0 +1,12 @@
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+  justify-content: center;
```
Intenta cambiar `justify-content: center;` para `justify-content:  space-around`!

misscs replied:
Checkout responsiveness now :D

misscs commented on app/views/scholarships/index.html:
```diff
@@ -1,9 +1,25 @@
 
 {% block page_content %}
+
   <h1>{{ scholarships.intro }}</h1>
```
Necesitamos este `h1`?

Clarissa0512 committed:
Añadiendo nueva sección para los estilos de las secciones macile.org

Aqui añado una sección nueva para los estilos de todas las secciones de macile.org

Clarissa0512 committed:
Removiendo el height.

Aquí remuevo la propiedad height.

Clarissa0512 committed:
Removiendo los margin innecesarios

Aquí remuevo margin-bottom, margin-right, margin-left para ponerlo todo en un solo margin.

Clarissa0512 committed:
Arreglando los nombres de las clases

Aquí arreglo los nombres e las clases tanto en html como en css para que vallan acorde con el nombre del macro.

Clarissa0512 committed:
Centralizando texto de header

Aquí centralizo el texto del header.

Clarissa0512 committed:
Removiendo los bordes de header

Aquí remuevo los bordes que tiene el header.

Clarissa0512 committed:
Indentando...

Aquí indento en los archivos de html.

Clarissa0512 committed:
Cambiando el nombre clase y del macro

Aquí cambio el nombre de las clases y del macro a "fact".

Clarissa0512 committed:
Cambiando formato del htlm

Aquí cambio el estilo del html y añado su estilo a css.

Clarissa0512 committed:
Poniendo el header flexible

Aquí pongo el header flexible usando yml.

Clarissa0512 committed:
Añadiendo los estilos de header

Aquí remuevo el comentario a los estilos del header

Clarissa0512 committed:
Cambiando la propiedad de justify-content

Aquí cambio la propiedad de justify-content de center a space-around.

Clarissa0512 committed:
Resolviendo conflictos en el archivo teaser.macro.html

Aqui resuelvo un conflicto en el archivo teaser.macro.html.

Clarissa0512 committed:
Remiviendo el width de la clase fact-content

Aquí remuevo el width de la clase fact-content

Clarissa0512 commented on app/modules/datos/datos.html:
```diff
@@ -0,0 +1,11 @@
+    </p>
+
+  </section>
+
```
Eliminar este espacio.

Clarissa0512 committed:
Merge branch 'master' into datos

 Conflicts:
	app/assets/styles/site.css
	app/modules/teaser_news/teaser.macro.html

Clarissa0512 committed:
Cambiando el nombre de la clase fact-header en css y html

Aqui le cambiamos el nombre al la clase fact-header por overview-header en css y html.

Clarissa0512 committed:
Colocando una division para facts

Aqui colocando facts dentro de una division y le damos estilos en css.

Clarissa0512 committed:
Removiendo espacio...

Aqui removemos un espacio innecesario.

Clarissa0512 committed:
Merge branch 'master' into datos

 Conflicts:
	app/assets/styles/site.css
	app/views/scholarships/index.html
